components/augeas/augeas-vim.p5m

```diff
@@ -0,0 +1,33 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-vim@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - ViM editor binding"
+set name=info.classification \
+    value=org.opensolaris.category.2008:Development/Editors \
+    value="org.opensolaris.category.2008:System/Text Tools"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# This binding needs ViM to be functional, but we don't want to require
+# installation of ViM if the end-user just installs the grouper package
+# and really only wants augtools, not the editor
+depend fmri=editor/vim	type=optional
```

Comment:
this dependency doesn't work - https://docs.oracle.com/cd/E53394_01/html/E54820/dependtypes.html#PKDEVglujw